src/qt/rpcconsole.cpp

```diff
@@ -706,6 +706,11 @@ void RPCConsole::setClientModel(ClientModel *model, int bestblock_height, int64_
 
         // create ban table context menu
         banTableContextMenu = new QMenu(this);
+        //: Context menu action to copy the subnet of a banned peer
+        banTableContextMenu->addAction(tr("Copy subnet"), [this] {
```

Comment:
Since #362 we are now defaulting to adding mnemonic shortcuts for the context menus. This means that you should add one here

Below is how you would do so here:
```suggestion
        banTableContextMenu->addAction(tr("&Copy subnet"), [this] {
```

Reply:
this is addressed in d518d67